lib/argsFor.js

```diff
@@ -40,7 +42,6 @@ exports.pack = function (nuspecFile, options) {
   });
 
   args.push("-nopackageanalysis");
-  args.push("-noninteractive");
```

Comment:
What is the reason to remove this? If we want it to be configurable it should probably default to run as `noninteractive` to keep backward compatability.

This is also causing the tests to fail.

Reply:
Totally correct that it should be backward compatible. I will update this.

We would need to make this a setting because of an authentication popup could popup when pushing to a secure nuget server. A great example is the Azure Devops Artifacts.